meta/main.yml

```diff
@@ -18,22 +18,23 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon Linux 2
-      versions:
-        - any
+    # Amazon Linux 2023 does not offer
+    # mod_auth_gssapi or mod_authnz_pam
```

Comment:
Can you further clarify this comment (and the others like it) to indicate that these packages are required to install httpd? Just to be crystal clear.

Reply:
@dav3r Please see 4fade50